### PR TITLE
fix: allow BitVector param type to Cryptol fns

### PR DESCRIPTION
**Pull request by pnwamk**
Should fix https://github.com/GaloisInc/argo/issues/110 - I've got a cryptol-remote-api test locally that successfully uses it, but nothing in this repo ... =\

### CODE REVIEW

File: python/cryptol/cryptoltypes.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def convert(self, val : Any) -> Any:
             return {'expression': 'bits',
                     'encoding': 'base64',
                     'width': val.length(), # N.B. original length, not padded
-                    'data': copy.get_bitvector_in_hex()}
+                    'data': base64.b64encode(bytes(int(val))).decode('ascii')}
```
Comment (david-christiansen) [inline python/cryptol/cryptoltypes.py:158]: I'm assuming that you've tested that this works round-trip. The sequence of conversions looks a bit mysterious :-)

Do we still need to pad the copy from the left if we're sending it as base64?

**Review by david-christiansen — CHANGES_REQUESTED** [review]

File: python/cryptol/cryptoltypes.py (hunk 1 above)
Comment (pnwamk) [reply python/cryptol/cryptoltypes.py]: I had it wrong. After some googling around, manual experiments, and a better test from the cryptol-remote-api package I think this is right after 90df637.

**Comment by pnwamk** [comment]
@david-christiansen if you think we need more testing I'm happy to add some somewhere, but things are looking good from manual poking and the corresponding BitVector test in the cryptol-remote-api PR here: https://github.com/GaloisInc/cryptol/pull/943

**Review by david-christiansen — APPROVED** [review]
This leaves us better off than before, at least :-)

**Pull request merged**